postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  plugins: {
+    tailwindcss: {},
```

Comment:
Just wondering, why was tailwindcss added to the configuration as well? Wasn't the fix only to add autoprefixer?